sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs

```diff
@@ -75,6 +80,7 @@ internal static string GetVersionString(ServiceVersion version)
             {
                 ServiceVersion.V3_0 => "v3.0",
                 ServiceVersion.V3_1 => "v3.1",
+                ServiceVersion.V3_2_Preview_1 => "v3.1",
```

Comment:
Don't want to break all tests now, so keeping the version as 3.1 internally. We'll need to change it once we want to add live tests.

Reply:
Could you create an issue for this?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/22800